specification/glossary.md

```diff
@@ -88,10 +88,6 @@ The suggested provider-published unit price for a single [Pricing Unit](#pricing
 
 The agreed-upon unit price for a single [Pricing Unit](#pricingunit) of the associated SKU, inclusive of negotiated discounts, if present, and exclusive of any other discounts. This price is denominated in the [Billing Currency](#glossary:billingcurrency).
 
-<a name="glossary:list-unit-price"><b>Contracted Unit Price</b></a>
```

Comment:
Can we also update the entry above with a corrected glossary name and fix any references?

`glossary:list-unit-price` --> `glossary:contracted-unit-price`

Reply:
+1 I noticed the same and was about to leave a comment 🙂

Reply:
Fixed. cc @jpradocueva

Reply:
I still see a couple outstanding links to `list-unit-price` ... 
<img width="971" alt="image" src="https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/assets/17853509/c592e41c-561a-4d93-813b-50528016dc21">